contrib/debian/copyright

```diff
@@ -171,10 +171,6 @@ License: CC0-1.0
 Comment: Other contributors are Simon Truscott (@bobbleclank), Kévin Alexandre Boissonneault (@KABoissonneault),
  and Björn Fahller (@rollbear).
 
-Files: depends/*/vendored-sources/orchard/*
-Copyright: 2020-2022 The Electric Coin Company
-License: BOSL-1+ with Zcash exception
```

Comment:
No replacement for this is needed because the policy is:
> We do not explicitly list Rust crates that have Expat/MIT as a licensing option (as determined by "cargo license").